src/main/kotlin/calculator/InputValidator.kt

```diff
@@ -0,0 +1,26 @@
+package calculator
+
+import calculator.Operator.Companion.isOperator
+
+class InputValidator {
+    fun validate(input: String) {
+        if (input.isBlank()) {
+            throw IllegalArgumentException("The input cannot be null or blank")
+        }
```

Comment:
kotlin에서 제공하는 [require 함수](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/require.html)를 활용해볼 수 있을 것 같아요.  😃
```suggestion
        require(input.isNotBlank()) { "The input cannot be null or blank" }
```